hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/consensus/TopicGetInfoSuite.java

```diff
@@ -58,7 +58,7 @@ private HapiSpec postCreateTopicCase() {
                                 .submitKeyName("submitKey")
                                 .autoRenewAccountId("autoRenewAccount"))
                 .when()
-                .then(getTopicInfo("testTopic")
+                .then(getTopicInfo("0.0.1")
```

Comment:
Need to revert if this is running in CI. 😛

Reply:
Ah yes 😛